added support for time field (#34094)
demisto/content: Packs/Elasticsearch/Integrations/Elasticsearch_v2/README.md

    | Client type | In some hosted ElasticSearch environments, the standard ElasticSearch client is not supported. If you encounter any related client issues, please consider using the OpenSearch client type. | False |
    | Index from which to fetch incidents (CSV) |  | False |
    | Query String | The query will be used when fetching incidents. Index time field will be used as a filter in the query | False |

>     | Index time field (for sorting sort and limiting data) | The time field on which sorting and limiting is done. If using nested field, separate fields name using a dot. | False |

Reviewer: ```suggestion
    | Index time field (for sorting sort and limiting data) | The time field on which sorting and limiting are performed. If using a nested field, separate field names using dot notation. | False |
```
